server/src/main/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplateServicesImpl.java

```diff
@@ -99,53 +82,22 @@ public FeedbackTemplate getById(UUID id) {
             throw new NotFoundException("No feedback template with ID " + id);
         }
 
-        if (!getIsPermitted()) {
-            throw new PermissionException("You are not authorized to do this operation");
-        }
 
         return feedbackTemplate.get();
     }
 
     @Override
     public List<FeedbackTemplate> findByFields(@Nullable UUID creatorId, @Nullable String title) {
-        if (!getIsPermitted()) {
```

Comment:
why did we remove this validation ?

Reply:
getIsPermitted only checks if there is an authenticated user. However, we determined that the backend throws an error if there is not an authenticated user before it even hits this service implementation, making it a moot point.